instances/treasury-factory.near/aliases.mainnet.json

```diff
@@ -4,6 +4,7 @@
   "REPL_BASE_DEPLOYMENT_ACCOUNT": "treasury-factory.near",
   "REPL_DEVDAO_ACCOUNT": "treasury-devdao.near",
   "REPL_SPUTNIK_FACTORY_ACCOUNT": "sputnik-dao.near",
+  "REPL_FACTORY_REFERENCE_ACCOUNT": "treasury-testing.near",
```

Comment:
Should we set up an account that would sound more like a production-ready app? `treasury-app.near`?

Reply:
@Megha-Dev-19 is working on this @frol